web/beacon-app/src/features/teams/components/ChangeRoleForm.tsx

```diff
@@ -0,0 +1,63 @@
+import { TextField } from '@rotational/beacon-core';
+import { Form, Formik, FormikHelpers } from 'formik';
+
+import Button from '@/components/ui/Button/Button';
+import Select from '@/components/ui/Select';
+
+import { ChangeRoleFormDto } from '../types/changeRoleFormDto';
+
+const ROLE_OPTIONS = [
+  { value: 'Owner', label: 'Owner' },
+  { value: 'Admin', label: 'Admin' },
+  { value: 'Observer', label: 'Observer' },
+  { value: 'Member', label: 'Member' },
```

Comment:
```suggestion
  { value: 'Observer', label: 'Observer' },
```

Reply:
I think having `Member` above `Observer` is better since a `Member` is ranked higher.